docs/data-sources/dbsnp.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: dbSNP
+---
+import JSON from './dbsnp-json.md'
+
+## VCF Extraction:
+
+### Example
+```
+1       10177   rs367896724     A       AC      .       .       RS=367896724;RSPOS=10177;dbSNPBuildID=138;SSR=0;SAO=0;VP=0x050000020005130026000200;GENEINFO=DDX11L1:100287102;WGT=1;VC=DIV;R5;ASP;G5A;G5;KGPhase3;CAF=0.5747,0.4253;COMMON=1;TOPMED=0.76728147298674821,0.23271852701325178
+```
+The following fields are extracted:
+- RS ID
+- Global minor allele
+- Global minor allele frequency
+- Global major allele
+- Global major allele frequency
+
+### Global allele extraction
+The global major and minor allele are extracted based on the frequency of the alleles provided in the CAF field. The global minor allele frequency is the second highest value of the CAF comma delimited field (ignoring '.' values). However, if there are two candidates for global major and the ref allele is one of them, we prefer the reference allele. On the other hand, if there are two candidates for global minor and the ref allele is one of them, we prefer the other alleles over the ref. If the ref allele is not involved, they are chosen arbitrarily . Here are a few examples to illustrate out policy:
+
+*Example 1:*
+
+```Chr1 100 A C CAF=0.5,0.5```
+
+we will select A to be the Global major and C to be the Global minor allele.
+
+*Example 2:*
+
+```Chr1 100 A C,T CAF=0.33,0.33,0.33```
+
+we will select A to be the Global major and either C or T is chosen (arbitrarily) to be the Global minor allele.
+
+*Example 3:*
+
+```Chr1 100 A C,T CAF=0.2,0.4,0.4```
+
+C or T will be arbitrarily assigned to be the Global major or minor allele.
+
+*Example 4:*
+
+```Chr1 100 A C,T CAF=0.2,0.2,0.6```
+
+Here T is the Global major allele while C is selected as the global minor.
+
+## Conflict resolution
```

Comment:
Move conflict resolution to the appropriate spot in the parsing section and use the remark notation:

:::caution Entries with Conflicting CAF values
If there are multiple entries with different CAF values for the same allele, we use the first CAF value.
:::

The other part is not part of conflict resolution and should be moved to another spot (and it's no longer true - we don't separate them with commas anymore)

Reply:
Yes, we have json array now.